cmd/neofs-node/config/fschain/chain.go

```diff
@@ -0,0 +1,44 @@
+package fschainconfig
+
+import (
+	"time"
+)
+
+const (
+	//DialTimeoutDefault is a default dial timeout of FS chain client connection.
+	DialTimeoutDefault = time.Minute
+	// CacheTTLDefault is a default value for cached values TTL.
+	// It is 0, because actual default depends on block time.
+	CacheTTLDefault = time.Duration(0)
+	// ReconnectionRetriesNumberDefault is a default value for reconnection retries.
```

Comment:
btw, it's `the default` everywhere, it's the one and only value here, not some abstract default.